test/e2e/scalenodes.go

```diff
@@ -96,16 +149,12 @@ func scale(name string, delta int32) error {
 
 func waitForScale(name string) error {
 	return wait.PollImmediate(10*time.Second, 30*time.Minute, func() (bool, error) {
-		ms, err := clients.MachineAPI.MachineV1beta1().MachineSets("openshift-machine-api").Get(context.Background(), name, metav1.GetOptions{})
+		ms, err := clients.MachineAPI.MachineV1beta1().MachineSets(machineSetsNamespace).Get(context.Background(), name, metav1.GetOptions{})
 		if err != nil {
 			log.Warn(err)
 			return false, nil // swallow error
 		}
 
-		if ms.Spec.Replicas == nil {
-			ms.Spec.Replicas = to.Int32Ptr(1)
```

Comment:
I removed this from `waitForScale` because it was not allowing the new test to zero out the MachineSets. It isn't important to the other test that uses `waitForScale`.